src/commands/config/mod.rs

```diff
@@ -27,13 +26,8 @@ pub fn global_config(user: &GlobalUser, verify: bool) -> Result<(), failure::Err
         validate_credentials(user)?;
     }
 
-    let toml = toml::to_string(&user)?;
-
-    let config_dir = get_global_config_dir().expect("could not find global config directory");
-    fs::create_dir_all(&config_dir)?;
-
-    let config_file = config_dir.join("default.toml");
-    fs::write(&config_file, &toml)?;
+    let config_file = default_config_file().expect("could not find global config directory");
```

Comment:
is there an actionable here or is this just a bad state? what should the user do to recover?

Reply:
that's a great question! atm this is copy 🍝